src/Tribe/REST/V1/Endpoints/Archive_Event.php

```diff
@@ -434,7 +434,7 @@ public function get_documentation() {
 	 *
 	 * @return array
 	 */
-	public function GET_args() {
+	public function READ_args() {
```

Comment:
I'm aware of this and took some time to consider if any aliasing would be in order here; I'd skip this as this is really a code niche and it helps sticking to WP verbiage standards.